rock.osdeps

```diff
@@ -251,3 +251,18 @@ ruby-backports:
 ruby-xmlrpc:
    gem: xmlrpc
 
+python:
```

Comment:
Not sure if Python has to be defined or whether it is already known to autoproj.

Reply:
Currently it is defined in rock package_set. We can remove it there, once the osdeps has been defined in rock.core.

rock/rock.osdeps
```
python:
    debian, ubuntu: python-dev
    opensuse: python-devel
```

Reply:
OK, should be fixed now...